File: FIL/FIL.Web.Admin/ClientApp/models/ReportingWizard/InventoryReportResponseDataViewModel.ts
import Transaction from "../Common/TransactionViewModel";
import TransactionDetail from "../Common/TransactionDetailViewModel";
import TicketCategory from "../Common/TicketCategoryViewModel";
import Event from "../Common/EventViewModel";
import EventDetail from "../Common/EventDetailViewModel";
import EventTicketDetail from "../Common/EventTicketDetailViewModel";
import EventTicketAttribute from "../Common/EventTicketAttributeViewModel";
import currencyType from "../Common/CurrencyTypeViewModel";
import Venue from "../Common/VenueViewModel";
import City from "../Common/CityViewModel";
import CurrencyType from "../Common/CurrencyTypeViewModel";
import CorporateTicketAllocationDetail from "../Common/CorporateTicketAllocationDetailViewModel";
import CorporateTransactionDetail from "../Common/CorporateTransactionDetailViewModel";
import Sponsor from "../Common/SponsorViewModel";
import ReportColumn from "../Common/ReportingColumn";

export class InventoryReportResponseDataViewModel {
    ticketCategory: TicketCategory[];
    eventTicketAttribute: EventTicketAttribute[];
    eventTicketDetail: EventTicketDetail[];
    corporateTicketAllocationDetail: CorporateTicketAllocationDetail[];
    corporateTransactionDetail: CorporateTransactionDetail[];
    sponsor: Sponsor[];
    transaction: Transaction[];
    transactionDetail: TransactionDetail[];
    reportColumns: ReportColumn[];
}

File: FIL.Web.Admin/ViewModels/Reporting/InventoryReportResponseDataViewModel.cs
using FIL.Contracts.Enums;
using FIL.Contracts.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace FIL.Web.Admin.ViewModels.Reporting
{
    public class InventoryReportResponseDataViewModel
    { 
        public IEnumerable<TicketCategory> TicketCategory { get; set; }
        public IEnumerable<EventTicketAttribute> EventTicketAttribute { get; set; }
        public IEnumerable<EventTicketDetail> EventTicketDetail { get; set; }
        public IEnumerable<CorporateTicketAllocationDetail> CorporateTicketAllocationDetail { get; set; }
        public IEnumerable<CorporateTransactionDetail> CorporateTransactionDetail { get; set; }
        public IEnumerable<Sponsor> Sponsor { get; set; }
        public IEnumerable<FIL.Contracts.Models.Transaction> Transaction { get; set; }
        public IEnumerable<TransactionDetail> TransactionDetail { get; set; }
        public IEnumerable<ReportingColumn> ReportColumns { get; set; }
    }
}
